bundles/model/org.eclipse.smarthome.model.core/src/main/java/org/eclipse/smarthome/model/core/internal/folder/FolderObserver.java

```diff
@@ -236,7 +236,8 @@ private static void checkFile(final ModelRepository modelRepo, final File file,
             try {
                 synchronized (FolderObserver.class) {
                     if ((kind == ENTRY_CREATE || kind == ENTRY_MODIFY)) {
-                        if (parsers.contains(getExtension(file.getName()))) {
+                        // we omit parsing of hidden files possibly created by editors or operating systems
+                        if (parsers.contains(getExtension(file.getName())) && !file.getName().startsWith(".")) {
```

Comment:
Just one more thing that crossed my mind: Why don't you simply use `file.isHidden()`? 
Then it would not only work on operating systems, but also on Windows 😉